pkg/v1/tkg/web/src/app/views/landing/azure-wizard/provider-step/azure-provider-step.component.spec.ts

```diff
@@ -146,85 +146,4 @@ describe('AzureProviderStepComponent', () => {
-    //         accessKeyID: "mykeyId",
-    //         region: "US-WEST",
-    //         secretAccessKey: "myKey",
-    //         sshKeyName: "myKeyName"
-    //     }));
-
-    //     component.ngOnInit();
-    //     fixture.whenStable().then(
-    //         () => {
-    //             fixture.detectChanges();
-    //             const connectBtn = fixture.debugElement.query(By.css("button.btn-primary"));
-    //             expect(connectBtn.nativeElement.disabled).toBeFalsy();
-    //         }
-    //     );
-    // }));
-
-    // it("should be disabled", async(() => {
-    //     mockedApiService.getAWSEndpoint.and.returnValue(of({
-    //         accessKeyID: "mykeyId",
-    //         region: "US-WEST",
-    //         secretAccessKey: "myKey",
-    //     }));
-
-    //     component.ngOnInit();
-    //     fixture.whenStable().then(
-    //         () => {
-    //             fixture.detectChanges();
-    //             const connectBtn = fixture.debugElement.query(By.css("button.btn-primary"));
-    //             expect(connectBtn.nativeElement.disabled).toBeTruthy();
-    //         }
-    //     );
-    // }));
-
-    // it("should be successful when clicked", async(() => {
-    //     mockedApiService.getAWSEndpoint.and.returnValue(of({
-    //         accessKeyID: "mykeyId",
-    //         region: "US-WEST",
-    //         secretAccessKey: "myKey",
-    //         sshKeyName: "myKeyName"
-    //     }));
-    //     mockedApiService.setAWSEndpoint.and.returnValue(empty());
-
-    //     const connectBtn = fixture.debugElement.query(By.css("button.btn-primary"));
-
-    //     connectBtn.nativeElement.click();
-    //     fixture.whenStable().then(
-    //         () => {
-    //             fixture.detectChanges();
-    //             const globalError = fixture.debugElement.query(By.css("app-step-form-notification"));
-    //             expect(globalError.nativeElement.innerText).toBe("");
-    //         }
-    //     );
-    // }));
-
-    // it("should show an error when clicked", async(() => {
-    //     mockedApiService.getAWSEndpoint.and.returnValue(of({
-    //         accessKeyID: "mykeyId",
-    //         region: "US-WEST",
-    //         secretAccessKey: "myKey",
-    //         sshKeyName: "myKeyName"
-    //     }));
-    //     mockedApiService.setAWSEndpoint.and.returnValue(throwError({ status: 400,
-    //         error: {message: 'failed to set aws endpoint' }}));
-
-    //     component.ngOnInit();
-    //     expect(component.errorNotification).toBeFalsy();
-
-    //     const connectBtn = fixture.debugElement.query(By.css("button.btn-primary"));
-    //     connectBtn.nativeElement.click();
-    //     fixture.whenStable().then(
-    //         () => {
-    //             fixture.detectChanges();
-    //             expect(component.errorNotification).not.toBeFalsy();
-    //             expect(component.validCredentials).toBeFalsy();
-    //         }
-    //     );
-    // }));
```

Comment:
If this is not needed, should we just remove this? Just wanted to confirm this is commented intentionally and not kept by mistake.

Reply:
Yes exactly, it's not super clear looking at the code review but my change is actually removing all of those unused tests